lib/model/imapmessage.php

```diff
@@ -479,6 +479,9 @@ public function getFullMessage($ownMail, $specialRole=null) {
 		$data = $this->getListArray();
 		if ($this->hasHtmlMessage) {
 			$data['hasHtmlBody'] = true;
+			if($specialRole === 'drafts'){
+				$data['body'] = rtrim($this->htmlMessage,"<br><br>");
```

Comment:
the root cause is `handleHtmlMessage()` function at line 571. i think `&lt;br&gt&lt;br&gt;` appending is needed. what is your opinion